serviceorchestration-simple/src/main/java/eu/arrowhead/serviceorchestration/BeanConfig.java

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
```

Comment:
This file uses spaces for indentation instead of tabs too. I assume all the files did in this module. This will be a problem in the future.